scripts/featured

```diff
@@ -26,7 +26,11 @@ PORT_INIT_TIMEOUT_SEC = 180
 
 def run_cmd(cmd, log_err=True, raise_exception=False):
     try:
-        subprocess.check_call(cmd)
+        result = subprocess.run(cmd,
```

Comment:
Using subprocess.run() i.s.o subprocess.check_call() lgtm, btw are we ok to dump the result.stdout, result.stderr in the syslog as INFO - will that be huge data 

Also it will be good to see why/how we got into this condition of "systemctl pmon enable" failing. Are we now trying to enable this service early via featured ?

Reply:
The stderr/stdout should not be huge but useful info returned by systemctl command to help any failure. In worst case we see the dump I listed in description when enable fail due to the service which does not have a WantBy directive.

systemctl pmon enable fails (and so do snmp, gnmi, lldp, etc) as these service can't not be enabled by systemctl as there is no WantBy directive. So systemctl just return the message suggesting same and does nothing. Its a no op.
These services are now started by featured using 'systemctl start <>' command each boot.

Reply:
pmon used to have WantedBy directive which was removed when pmon.timer was added by https://github.com/sonic-net/sonic-buildimage/commit/4ec3af86af5c9ed2da21df33111fa84181069a3e
After this timer used to trigger pmon.
Later timer was removed to introduce featured
so pmon is not enabled any more for systemd to start it.
featured starts it with 'systemctl start'
'systemctl enable' is a no op here now